documentation/docs/guides/options.md

```diff
@@ -27,13 +27,13 @@ func main() {
 
 ## Some options
 
-### Port
+### Address
```

Comment:
We still need to explicitely mention Port for people searching for "port" in the docs.

```suggestion
### Address (and port)
```

Reply:
Good point